app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,27 +1,37 @@
 import React, { useState, useEffect } from "react";
 
-import EmptyNotesListImage from "images/EmptyNotesList";
 import { Button, PageLoader } from "neetoui";
 import { Container, Header } from "neetoui/layouts";
 
 import notesApi from "apis/notes";
-import EmptyState from "components/Common/EmptyState";
-import { useUserState } from "contexts/user";
 
 import DeleteAlert from "./DeleteAlert";
 import NotesList from "./NotesList";
 import NotesMenu from "./NotesMenu";
 import NewNotePane from "./Pane/Create";
+import EditNotePane from "./Pane/Edit";
 
 const Notes = () => {
-  const { user } = useUserState();
   const [loading, setLoading] = useState(true);
+  const [showNotesMenu, setshowNotesMenu] = useState(false);
   const [showNewNotePane, setShowNewNotePane] = useState(false);
+  const [showEditNote, setShowEditNote] = useState(false);
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
-  const [searchTerm, setSearchTerm] = useState("");
-  const [selectedNoteIds, setSelectedNoteIds] = useState([]);
+
   const [notes, setNotes] = useState([]);
-  const [showNotesMenu, setshowNotesMenu] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+  const [selectedNote, setSelectedNote] = useState({});
+  const [selectedNoteId, setSelectedNoteId] = useState([]);
```

Comment:
Not sure why we need two states here to handle same entity. So when we set `selectedNote`, aren't we getting the `id` of that note too?